UI/src/view/point_asset/obstacleForm.js

```diff
@@ -28,7 +32,8 @@
         '        <option value="1" '+ (parseInt(me.selectedAsset.getByProperty('esterakennelma')) === 1 ? 'selected' : '') +'>Suljettu yhteys</option>' +
         '        <option value="2" '+ (parseInt(me.selectedAsset.getByProperty('esterakennelma')) === 2 ? 'selected' : '') +'>Avattava puomi</option>' +
         '      </select>' +
-        '    </div>';
+        '    </div>' +
+          components;
     };
 
     this.boxEvents = function(rootElement, selectedAsset, localizedTexts, authorizationPolicy, roadCollection, collection) {
```

Comment:
The unused function warning is appearing because there is no direct call for this method in the current asset. However this will be call by it's parent in PointAssetForm